1-WebApp-OIDC/1-6-SignOut/README.md

```diff
@@ -65,13 +65,10 @@ public static IServiceCollection AddAzureAdV2Authentication(this IServiceCollect
         ...
         options.Authority = options.Authority + "/v2.0/";
         ...
-        // Attempt to avoid displaying the select account dialog when signing out
+        
         options.Events.OnRedirectToIdentityProviderForSignOut = async context =>
         {
```

Comment:
I'd suggest removing the doc bits completely as well. But I'll let others weigh in as well.

Reply:
snap :)